units/BROT1BTT2/BROT1BTT2_Script.lua

```diff
@@ -57,6 +57,10 @@ BROT1BTT2 = Class(AHoverLandUnit){
 		},
 		clawgun = Class(TDFHeavyPlasmaCannonWeapon){ FxMuzzleFlashScale = 0.4 },
 	},
+	OnCreate = function(self)
+		AHoverLandUnit.OnCreate(self)
+		self.Blueprint.Display.BuildMeshBlueprint = false
```

Comment:
This is not exactly a solution. Wouldn't it be better to investigate the actual cause?